package.json

```diff
@@ -72,6 +72,7 @@
     "@babel/register": "^7.0.0",
     "@changesets/cli": "^2.27.11",
     "@ebay/skin": "^18.4.0",
+    "@floating-ui/react": "^0.27.3",
```

Comment:
@HenriqueLimas why on dev dependencies? you need it as part of the component right?

Reply:
It should be dependency, thanks let me update